src/Monolog/Logger.php

```diff
@@ -176,6 +176,25 @@ public function popHandler()
     }
 
     /**
+     * Set handlers, removing all existing ones.
+     * Falsey values will be ignored, and if a map is passed, keys will be ignored.
```

Comment:
why ignoring falsey values ? It does not make snese IMO, especially given what you document below saying that all elements must be of type HandlerInterface. If you respect this, you cannot pass a falsey value